desktop/src/navigator.rs

```diff
@@ -160,10 +161,20 @@ impl NavigatorBackend for ExternalNavigatorBackend {
                 let client =
                     client.ok_or_else(|| Error::FetchError("Network unavailable".to_string()))?;
 
-                let isahc_request = match request.method() {
+                let mut isahc_request = match request.method() {
                     NavigationMethod::Get => IsahcRequest::get(processed_url.to_string()),
                     NavigationMethod::Post => IsahcRequest::post(processed_url.to_string()),
                 };
+                if let Some(headers) = isahc_request.headers_mut() {
+                    for (name, val) in request.headers().iter() {
+                        headers.insert(
```

Comment:
Per the docs, it might be that [`append`](https://docs.rs/http/latest/http/header/struct.HeaderMap.html#method.append) is more suitable than [`insert`](https://docs.rs/http/latest/http/header/struct.HeaderMap.html#method.insert), as the former allows multiple headers of the same name.

Reply:
Testing shows that flash player only considers the last header in the array for a given name (later values overwrite earlier values). I'll add a comment explaining that.